flask_admin/model/base.py

```diff
@@ -1242,11 +1291,45 @@ def _create_ajax_loader(self, name, options):
         raise NotImplementedError()
 
     # Views
-    @expose('/')
+    @expose('/', methods=('POST', 'GET'))
```

Comment:
Quick question: form is submitted via AJAX POST or a normal form POST?

If it is AJAX POST, it is better to use separate endpoint. If it is normal post - can we use AJAX POST?